gemrb/core/GUI/WindowManager.cpp

```diff
@@ -41,7 +42,8 @@ Holder<Sprite2D> WindowManager::CursorMouseDown;
 
 void WindowManager::SetTooltipDelay(int delay)
 {
-	ToolTipDelay = delay;
+	// Max setting disables tooltips in originals, we just set it to an extremely high value for simplicity.
+	ToolTipDelay = (delay == 10 * Tooltip::DELAY_FACTOR) ? 1e7 : delay;
```

Comment:
Please just write the number out, which will also avoid a pointless cast.

Reply:
Like this?